examples/client_token_auth/client_token_auth.js

```diff
@@ -0,0 +1,28 @@
+const mc = require('minecraft-protocol')
+
+if (process.argv.length < 3 || process.argv.length > 5) {
+  console.log('Usage : node echo.js <host> <port> [<name>]')
+  process.exit(1)
+}
+
+const client = mc.createClient({
+  host: process.argv[2],
+  port: parseInt(process.argv[3]),
+  username: process.argv[4] ? process.argv[4] : 'echo'
```

Comment:
better remove the default to echo, this example doesn't make sense in offline mode